### PR TITLE
Added the spot usage of linked accounts to payer dashboard

### PR DESCRIPTION
**Pull request by athiruma**
1. Added the spot usage
2. Add the savings plans operations for the coming tasks

### CODE REVIEW

File: cloud_governance/common/clouds/aws/cost_explorer/cost_explorer_operations.py [hunk 1]
```diff
@@ -52,6 +52,36 @@ def __get_ce_tag_filters(self, tag: str, ce_default_filter: dict):
                 ce_default_filter = ce_filter
         return ce_default_filter
 
+    def get_ce_report_filter_data(self, ce_response: dict, tag_name: str):
+        """
+        This method return data filter by tag_name
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/cost_explorer/cost_explorer_operations.py:57]: This method returns

File: cloud_governance/common/clouds/aws/savingsplan/savings_plans_operations.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+from datetime import datetime
+
+import boto3
+from dateutil.relativedelta import relativedelta
+
+
+class SavingsPlansOperations:
+
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/savingsplan/savings_plans_operations.py:8]: Missing Documentation

File: cloud_governance/policy/aws/cost_explorer_payer_billings.py [hunk 3]
```diff
@@ -172,9 +179,35 @@ def get_linked_accounts_usage(self):
                 account = usage['Account']
                 cost_usage_data.setdefault(account, {}).update({idx: usage})
         self.get_linked_accounts_forecast(linked_account_usage=cost_usage_data)
+        self.__get_ce_cost_usage_by_filter_tag(tag_name='spot', cost_centers=cost_centers, cost_usage_data=cost_usage_data)
+        handler.setLevel(logging.WARN)
```
Comment (ebattat) [inline cloud_governance/policy/aws/cost_explorer_payer_billings.py:183]: Add document why we do this hack WARN - INFO

File: cloud_governance/policy/aws/cost_explorer_payer_billings.py [hunk 4]
```diff
@@ -172,9 +179,35 @@ def get_linked_accounts_usage(self):
...
         self.upload_data_elastic_search(linked_account_usage=cost_usage_data)
+        handler.setLevel(logging.INFO)
         return cost_usage_data
 
+    def __get_ce_cost_usage_by_filter_tag(self, cost_centers: list, tag_name: str, cost_usage_data: dict):
+        """
+        This method return the cost by filter tag_name
```
Comment (ebattat) [inline cloud_governance/policy/aws/cost_explorer_payer_billings.py:190]: returns

**Review by ebattat — COMMENTED** [review]
Do we need to add some tests ?

**Comment by athiruma** [comment]
> Do we need to add some tests?

No, We already added tests for the cost explorer service

**Pull request merged**